impeller/core/capture.cc

```diff
@@ -117,16 +117,16 @@ void Capture::Rewind() {
 #ifdef IMPELLER_ENABLE_CAPTURE
 #define _CAPTURE_PROPERTY_RECORDER_DEFINITION(type_name, pascal_name,          \
                                               lower_name)                      \
-  type_name Capture::Add##pascal_name(const std::string& label,                \
-                                      type_name value,                         \
+  type_name Capture::Add##pascal_name(std::string_view label, type_name value, \
                                       CaptureProperty::Options options) {      \
     if (!active_) {                                                            \
       return value;                                                            \
     }                                                                          \
     FML_DCHECK(element_ != nullptr);                                           \
                                                                                \
+    std::string label_clone = std::string(label);                              \
```

Comment:
Can we not push the use of string_view further into the stack so callers don't have to make copies?

Reply:
We could. I decided to stop here because the rest of this is debug only is compiled out for profile/release modes.